service/matching/pri_task_reader.go

```diff
@@ -510,7 +510,7 @@ func (tr *priTaskReader) shouldGCLocked() bool {
 }
 
 // called in new goroutine
-func (tr *priTaskReader) doGC() {
+func (tr *priTaskReader) doGC(ackLevel int64) {
```

Comment:
have to actually use this in the code below and move the db call outside the lock?